### PR TITLE
[MB-4524]  Job Runner: Approved Payment Request Fetcher

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description
This PR is the first code addition to creating a job runner to handle payment request that have been approved by the TIO and need to be sent along to Syncada via GEX. This PR specifically is focused on creating a method on the job runner that queries for all payment request that have a status of reviewed 

## Setup/Review Notes

The job runner isn't fully complete at this point. For this PR please review the query and unit test to ensure they are correct

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4524) for this change
* [Job Runner Proposal](https://docs.google.com/document/d/1p6LqIo7kRTqhKy_nqbnJEz8MsNowPiKqMlehAbDBElg/edit#) explains more about the approach used.

### CODE REVIEW

File: pkg/services/invoice/edi_job_runner.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	"github.com/transcom/mymove/pkg/models"
+	"github.com/transcom/mymove/pkg/services"
+)
+
+type ghcPaymentRequestJobRunner struct {
+	db *pop.Connection
+}
+
+// NewGHCJobRunner returns an implementation of the GHCJobRunner interface
+func NewGHCJobRunner(db *pop.Connection) services.GHCJobRunner {
```
Comment (jacquelineIO) [inline pkg/services/invoice/edi_job_runner.go:17]: Hey @pearl-truss  I feel like good instructions were not given for this story. The work is good, but we should remove the name `NewGHCJobRunner` we aren't implementing that just yet and I think the user story is not worded the best. 

This story should be to create a function/service very similar to 

```
// NewPaymentRequestFetcher returns a new payment request fetcher
func NewPaymentRequestFetcher(db *pop.Connection) services.PaymentRequestFetcher {
	return &paymentRequestFetcher{db}
}

//FetchPaymentRequest finds the payment request by id
func (p *paymentRequestFetcher) FetchPaymentRequest(paymentRequestID uuid.UUID) 
...
```

found under `pkg/services/payment_request`. Create a new file here.

**Review by jacquelineIO — CHANGES_REQUESTED** [review]
The gist of the work is good. Need to move the files around to line with with our other fetchers.

File: pkg/services/invoice/edi_job_runner.go (hunk 1 above)
Comment (pearl-truss) [reply pkg/services/invoice/edi_job_runner.go]: Thanks for looking at this! I think I have everything right side up now

**Review by jacquelineIO — APPROVED** [review]
thanks for the changes

**Pull request merged**